nodeApps/createPrivateRelease.sh

```diff
@@ -88,17 +112,25 @@ echo "Latest commit is: ${CURRENT_SHA}"
 # Tag this deploy and make a release in github
 RELEASE_JSON="{\"tag_name\": \"v${THIS_VERSION}\",\"target_commitish\": \"${CURRENT_SHA}\",\"name\": \"v${THIS_VERSION}\",\"body\": \"${MESSAGE}\",\"draft\": false,\"prerelease\": false}"
```

Comment:
`$(MESSAGE}` doesn't include `${CIRCLE_USERNAME}` which is really useful for seeing who deployed a given release, can we add that in please.

e.g. https://github.com/holidayextras/revolver/releases/tag/v6.29.0